consul/install.sls

```diff
@@ -62,18 +77,3 @@ consul_config_directory:
     - require_in:
       - file: configure_consul_service
```

Comment:
Shouldn't this be removed?

Reply:
I think it's worth keeping there since we are still going to be defaulting to installing Consul and won't likely _not_ install it with this formula.